common/ux-window.cpp

```diff
@@ -582,7 +592,7 @@ namespace rs2
         while (res && (!_app_ready || _splash_timer.get_elapsed_ms() < 2000.f))
         {
             res = !glfwWindowShouldClose(_win);
-            glfwPollEvents();
+           // glfwPollEvents();
```

Comment:
Not needed?

Reply:
I think its redundant works with and without it ,not effecting the Splash Screen
 I'm not sure if to remove it